src/panels/lovelace/hui-unused-entities.ts

```diff
@@ -95,13 +165,43 @@ export class HuiUnusedEntities extends LitElement {
           entities: domains[domain].map((entity) => ({
             entity,
             secondary_info: "entity-id",
+            type: this.lovelace!.editMode ? "select" : "default",
           })),
           show_header_toggle: false,
         });
         el.hass = this._hass;
+        if (this.lovelace!.editMode) {
+          el.addEventListener("entity-selection-changed", (ev: any) =>
+            this._handleSelectionChanged(ev)
+          );
+        }
         return el;
       });
   }
+
+  private _handleSelectionChanged(ev: any): void {
+    if (ev.detail.selected) {
+      this._selectedEntities.push(ev.detail.entity);
+    } else {
+      const index = this._selectedEntities.indexOf(ev.detail.entity);
+      if (index !== -1) {
+        this._selectedEntities.splice(index, 1);
+      }
+    }
+  }
+
+  private _selectView(): void {
+    showSelectViewDialog(this, { lovelace: this.lovelace! });
+    window.addEventListener("view-selected", (e) => this._addCard(e));
```

Comment:
Not happy with this, suggestions?

Reply:
yeah that's not good. Pass a callback to the dialog params instead.

Reply:
Like this https://github.com/home-assistant/home-assistant-polymer/blob/0008a100f45acd46a21b8477aa17f9dd6b95822f/src/dialogs/config-flow/show-dialog-data-entry-flow.ts#L73-L78